addons/mail/static/src/core/web/mail_core_web_service.js

```diff
@@ -35,11 +36,12 @@ export class MailCoreWeb {
                 );
             });
         });
+        this.busService.start();
```

Comment:
Before this commit, `busService.start()` was called after `isReady()` is resolved.
With this commit, this is called before `isReady()` is called.
This is a difference of behaviour that doesn't seem to be intended with this PR, so make sure to do the same as before, which was `messaging.isReady.then(() => this.busService.start())`

This comment applies to all positive `this.busService.start()` in this PR